templates/base.html

```diff
@@ -2,20 +2,76 @@
 
 <!DOCTYPE html>
 <html lang="en-gb">
-<meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-<head>
+  <head>
     <title>{% block title %} {% endblock %}| Share Spots</title>
```

Comment:
Just looked at this one and wonder if our app is `Sharespots` or `Share Spots`?

Reply:
Looking at the [designs](https://www.figma.com/file/LcuMvUJmLSrCEB0Qy3D4BDIM/Sharespots?node-id=0%3A1) it's `Sharespots`